fr.handbrake.ghb.json

```diff
@@ -0,0 +1,147 @@
+{
+    "app-id": "fr.handbrake.ghb", 
+    "modules": [
+        {
+            "name": "handbrake", 
+            "modules": [
+                {
+                    "sources": [
+                        {
+                            "url": "http://www.nasm.us/pub/nasm/releasebuilds/2.13.02/nasm-2.13.02.tar.xz", 
+                            "sha256": "8ac3235f49a6838ff7a8d7ef7c19a4430d0deecc0c2d3e3e237b5e9f53291757", 
+                            "type": "archive"
+                        }
+                    ], 
+                    "cleanup": [
+                        "*"
+                    ], 
+                    "name": "nasm"
+                }
+            ], 
+            "config-opts": [
+                "--flatpak", 
+                "--disable-gtk-update-checks"
+            ], 
+            "sources": [
+                {
+                    "url": "https://download.handbrake.fr/releases/1.1.1/HandBrake-1.1.1-source.tar.bz2", 
+                    "sha256": "e3390c5fd901fb06d72e29c62a63d373d5fb5b3467295d114d815ae7b78a9d7a", 
+                    "type": "archive", 
+                    "strip-components": 1
+                }, 
+                {
+                    "commands": [
+                        "mkdir -p download"
+                    ], 
+                    "type": "shell"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/jansson-2.10.tar.bz2", 
+                    "sha256": "241125a55f739cd713808c4e0089986b8c3da746da8b384952912ad659fa2f5a", 
+                    "type": "file", 
+                    "dest-filename": "download/jansson-2.10.tar.bz2"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/libbluray-1.0.2.tar.bz2", 
+                    "sha256": "6d9e7c4e416f664c330d9fa5a05ad79a3fb39b95adfc3fd6910cbed503b7aeff", 
+                    "type": "file", 
+                    "dest-filename": "download/libbluray-1.0.2.tar.bz2"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/lame-3.100.tar.gz", 
+                    "sha256": "ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e", 
+                    "type": "file", 
+                    "dest-filename": "download/lame-3.100.tar.gz"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/yasm-1.3.0.tar.gz", 
```

Comment:
yasm is included in the runtime.

Reply:
It's installed in a "contrib" tools subdirectory of the HandBrake build directory.  If the HandBrake build directory is not part of the runtime, neither is yasm.

Reply:
Sorry, i misunderstood your remark.  

I'm not sure why I'm building yasm here.  Perhaps it wasn't in an older version of the runtime (I started experimenting with this nearly 2 years ago). It can be removed in a future release.